spectacles/validators/content.py

```diff
@@ -22,6 +31,32 @@ def _get_personal_folders(self) -> List[int]:
                 personal_folders.append(folder["id"])
         return personal_folders
 
+    def _get_all_subfolders(self, folders: List[int]) -> List[int]:
+        result = []
+        for folder in folders:
+            result.extend(self._get_subfolders(folder))
+        return result
+
+    def _get_subfolders(self, folder_id: int) -> List[int]:
+        all_folders = self.client.all_folders(self.project.name)
+        subfolders = [folder_id]
+        children = [
+            child["id"] for child in all_folders if child["parent_id"] == folder_id
+        ]
+        if children:
+            for child in children:
+                subfolders.extend(self._get_subfolders(child))
+        return subfolders
+
+    def _ignore_folder_result(self, folder_id: Optional[str]) -> bool:
```

Comment:
I like the `is_*` naming convention for bool-returning functions. What about `is_selected`? That would be consistent with the way we describe this sort of thing for explores and models, plus it makes the if statement cleaner in `validate` (no continue needed).